docker/bigdl/deployment-k8s.yaml

```diff
@@ -16,10 +16,10 @@ spec:
         appType: bigdl
     spec:
       containers:
-        - image: intelanalytics/bigdl:default
+        - image: intelanalytics/bigdl-spark-3.1.2:ctc-client-1.0
```

Comment:
why ctc? does default work?

Reply:
The `default` tag is now gone, and the `ctc-client-1.0` is currently an available image